resources/js/views/Recipes.vue

```diff
@@ -160,6 +163,24 @@
         this.editedIndex = -1;
       }, 300);
     },
+
+    download() {
+        if (this.editedIndex > -1) {
+            Object.assign(this.recipes[this.editedIndex], this.editedItem);
+            this.snackbarText = "Recipe Downloaded";
+            this.snackbar = true;
+        
+          var vm= this,
+          blob = new Blob(['Recipe Name: ', vm.editedItem.name,' Category: ',this.editedItem.category.name], { type: 'text/plain' }),
+          anchor = document.createElement('a');
+      
+          anchor.download = _.kebabCase(vm.editedItem.name)+'.txt';
+          anchor.href = (window.webkitURL || window.URL).createObjectURL(blob);
+          anchor.dataset.downloadurl = ['text/plain', anchor.download, anchor.href].join(':');
+          anchor.click();
```

Comment:
maybe refactor this out to its own method, tbh not a big deal though it can stay.